src/Model/BlogObject.php

```diff
@@ -83,7 +83,7 @@ public function validate()
     public function getLink()
     {
         return Controller::join_links(
-            $this->Blog()->Link(),
+            $this->Blog()->Link(true),
             $this->getListUrlSegment(),
             $this->URLSegment
         );
```

Comment:
This does work, but it relies on a pretty weak `falsy` check against a parameter that's meant to be `string|null` - passing a boolean isn't really _supposed_ to be done.
Passing the arguments as the "action" seems like a more reliable way to go.

```suggestion
        return $this->Blog()->Link(Controller::join_links(
            $this->getListUrlSegment(),
            $this->URLSegment
        ));
```

Reply:
adapted as requested